releasenotes/notes/network-path-improve-packet-parsing-51b743b324f85764.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Improve parsing performance for packets coming in that are being read
```

Comment:
(nit) I think we don't need release note for this kind of change (parser perf improvement)

The CI should pass fine if you add the label changelog/no-change